docs/world api.md

```diff
@@ -490,6 +490,9 @@ In addition, the following methods can be implemented and are called in this ord
   after this step. Locations cannot be moved to different regions after this step.
 * `set_rules(self)`
   called to set access and item rules on locations and entrances.
+* `connect_entrances(self)`
+  by the end of this step, all entrances must exist and be connected to their source and target regions.
+  Entrance randomization should be done here.
```

Comment:
Probably a good idea to specify why it should be done here as opposed to "here or earlier" (ie to be able to handle plando)

Reply:
Well, that's not true yet though
In the GER docs I explain that soon:tm:, plando and GER will work together and this will rely on that specific timing
I guess I could copy that here too

Reply:
Oh true, brain fart, I think we can leave as-is